appsmithorg/appsmith
# Adding zoom on image widget.

## Description
> Adds zoom feature to Image widget. There is a new property named `maxZoomLevel` in ImageWidget. By default the maxZoomLevel is 1. It determines the max possible zoom and whether the zoom will be enabled. If the `maxZoomLevel` is greater than 1, zoom is enabled.

## Type of change
- New feature (non-breaking change which adds functionality)
- This change requires a documentation update

## How Has This Been Tested?

- Set `maxZoomLevel` as mentioned in description and check.
- Check drag drop and scroll inside and outside Image widget.

## Checklist:
